Library/Homebrew/config.rb

```diff
@@ -39,7 +39,7 @@
 HOMEBREW_LOGS = Pathname.new(ENV["HOMEBREW_LOGS"] || "~/Library/Logs/Homebrew/").expand_path
 
 # Must use /tmp instead of $TMPDIR because long paths break Unix domain sockets
-HOMEBREW_TEMP = Pathname.new(ENV.fetch("HOMEBREW_TEMP", "/tmp"))
+HOMEBREW_TEMP = Pathname.new(ENV["HOMEBREW_TEMP"]).realpath
```

Comment:
I think this will crash when the dir doesn't exist.

Reply:
Yup. Fixing 😭

Reply:
https://github.com/Homebrew/brew/pull/4415

Reply:
Thanks!